README.md

```diff
@@ -19,15 +19,32 @@ during a NANOG meeting that aimed to promote the usage of the iCalendar format.
+
+- **provider**: identifies the provider of the service that is the subject of the maintenance notification.
+- **account**: identifies an account associated with the service that is the subject of the maintenance notification.
+- **maintenance_id**: contains text that uniquely identifies the maintenance that is the subject of the notification.
+- **circuits**: list of circuits affected by the maintenance notification and their specific impact.
+- **status**: defines the overall status or confirmation for the maintenance.
+- **start**: timestamp that defines the start date of the maintenance in GMT.
+- **end**: timestamp that defines the end date of the maintenance in GMT.
+- **stamp**: timestamp that defines the update date of the maintenance in GMT.
+- **organizer**: defines the contact information included in the original notification.
+
+> Please, refer to the [BCOP](https://github.com/jda/maintnote-std/blob/master/standard.md) to more details about these attributes.
 
 ## Workflow
 
 1. We instantiate a `Provider`, directly or via the `init_provider` method, that depending on the selected type will return the corresponding instance.
-2. Each `Provider` have already defined multiple `Processors` that will be used to get the `Maintenances` when the `Provider.get_maintenances(data)` method is called.
-3. Each `Processor` class can have a pre defined logic to combine the data extracted from the notifications and create the final `Maintenance` object, and receives a `List` of multiple `Parsers` that will be to `parse` each type of data.
-4. Each `Parser` class supports one or more data types and implements the `Parser.parse()` method used to retrieve a `Dict` with the relevant key/values.
-5. When calling the `Provider.get_maintenances(data)`, the `data` argument is an instance of `NotificationData` (which is just a collection of multiple `DataParts`, each one with a `type` and a `content`) that will be used by the corresponding `Parser` when the `Processor` will try to match them.
+2. Get an instance of the `NotificationData` class that groups together `DataParts` which contain some content and a specific type (that will match a specific `Parser`), and adding some factory methods to initialize it from a single content (as before) or directly from a raw email content or `email.message.EmailMessage` instance.
+3. Each `Provider` have already defined multiple `Processors` that will be used to get the `Maintenances` when the `Provider.get_maintenances(data)` method is called.
+4. Each `Processor` class can have a custom logic to combine the data extracted from the notifications and create the final `Maintenance` object, and receives a `List` of multiple `Parsers` that will be to `parse` each type of data.
```

Comment:
```suggestion
4. Each `Processor` class uses one or more `Parsers` to process each type of data that it handles. It can have custom logic to combine the parsed data from multiple `Parsers` to create the final `Maintenance` object.
```